cli/cmd/install_test.go

```diff
@@ -187,7 +187,7 @@ func TestRender(t *testing.T) {
 	}
 }
 
-func testInstallOptions() (*installOptions, error) {
+func testInstallOptions(addCertDataOptions bool) (*installOptions, error) {
```

Comment:
what about always including the certDataOptions but manually unsetting fields in the returned options when necessary?